src/platforms/flutter/integrations/routing-instrumentation.mdx

```diff
@@ -63,19 +63,50 @@ final router = GoRouter(
 ### 2. Define Route Names
 
 By default the application's main route name is `"/"`.
-For transactions to be created automatically when navigation changes, you need to provide route names through your routes settings.
 The instrumentation sets the span `operation` to `navigation` and the span `name` to the provided route name.
+For transactions to be created when navigation changes, you need to provide route names:
 
-```dart
+- GoRouter: use the `name` parameter to set the route name.
+- Flutter routing: use `RouteSettings` and set the name in the constructor.
```

Comment:
Consider ordering these bullets in the same tab order as the code samples

```suggestion
- Flutter routing: use `RouteSettings` and set the name in the constructor.
- GoRouter: use the `name` parameter to set the route name.
```

Reply:
thx 🙏